kubernetes/cmsweb/scripts/deploy-secrets.sh

```diff
@@ -18,14 +18,15 @@ fi
 mkdir -p $tmpDir
 cd $tmpDir
 
-if [ -z "$(command -v sops)" ]; then
+SOPS=$(command -v sops)  || {
     # download soap in tmp area
     wget -O sops https://github.com/mozilla/sops/releases/download/v3.7.2/sops-v3.7.2.linux.amd64
     chmod u+x sops
     mkdir -p $HOME/bin
     echo "Download and install sops under $HOME/bin"
     cp ./sops $HOME/bin
-fi
+    SOPS="$HOME/bin/sops"
+}
```

Comment:
I suggest to add additional test here something like this:
```
if [ ! -f $SOPS ]; then
   echo "$SOPS does not exist"
   exit 1
fi
```
The point is that we should verify existence of the file. Moreover, I suggest to run  to run `$SOPS --version` and apply some regexp pattern matching to verify that it is indeed sops executable with version which we expect.

Reply:
which is the expected  version ?